File: Edinburgh College/JS/4. Arrays/activity2.js
// Sample test data
const sample_array = [20, 30, 25, 35, -16, 60, -100];
let add_data = 0;   // Initialise variable
// Sum data in array, looping each element
for (let i = 0; i < sample_array.length; i++) {
    add_data += sample_array[i];
}
let average = add_data / sample_array.length;
console.log(`Average value of the array elements is: ${average.toFixed(1)}.`);

File: Edinburgh College/Python/4. Python Collections/activity2.py
# Average of list

# Sample list
sample_list = [20, 30, 25, 35, -16, 60, -100]

# Initialise total
total = 0

# Loop through list
for number in sample_list:
    total += number  # Add number to total

# Get average
average = total / len(sample_list)

print("Average value of the list elements is:", round(average, 1))
